core/src/main/scala/spinoco/fs2/cassandra/util/util.scala

```diff
@@ -49,14 +49,13 @@ package object util {
     go(0,"")
   }
 
-  implicit class ThrowableRethrowSyntax(self: Throwable){
-    def rethrow(format: String, str: String*): Throwable = {
-      val msg = String.format(format, str:_*)
-      new Throwable(msg + s"\n    got: ${self.getMessage}")
+  object ExceptionWrapper {
+    def fromStmt(err: Throwable, stmt: String): Throwable = {
```

Comment:
I would do it that way, but both the field and statement are already strings in those parts of code. Instead of the field I may pass in the witness from which the string is infered, but that is hardly more descriptive....